### DISCUSSION TITLE
首页轮播图更换时候遇到问题

### DISCUSSION THREAD

**Discussion by SuperJeason** [theme-shoka-x/hexo-theme-shokaX#295] [Q&A] [answered]
大佬你好，冒昧打扰您一下，在使用shokaX主题时，根据大佬的教程文档在source/_data下建立了images_index.yml文件，但是首页轮播图并没有变化，主题文件夹下我并没有找到_images_index.yml，请教一下可能是哪里的问题，大佬可以帮忙看看首页轮播图这个图片储存在哪里嘛(SuperJeason.github.io)

**Comment by FidStyle** [comment, marked as answer]
#296
